packages/rancher-monitoring/rancher-pushprox/package.yaml

```diff
@@ -1,2 +1,3 @@
 url: local
-version: 103.0.0
\ No newline at end of file
+version: 104.0.0
+doNotRelease: true
```

Comment:
why is this package now marked `doNotRelease`?

Reply:
During the last branching push the `doNotRelease` was removed but it probably shouldn't have been. Pushprox is not meant to be used on its own and shouldn't be released on its own